process/rating/nilListIndexUpdater.go

```diff
@@ -0,0 +1,15 @@
+package rating
+
+// NilListIndexUpdater is a nil implementation for an list index updater
+type NilListIndexUpdater struct {
```

Comment:
what are we using the nilListindexUpdater for right now?
If this is not meant for production code but for testing, I would prefer if this was a mock instead and in mock folder.

Reply:
it is not meant for testing. It's the usage of `null object design pattern`. It follows the same direction for rating reader because these 2 components are initialized "lazy" inside validator statistics constructor. Will rename both of them with `disabled` instead of `nil`